liferay-tango/liferay-portal
# LPS-143214 Add id information to content dashboard autogenerated xls file

**Motivation**
----------------

Bug fixing:  id of each element present in the content dashboard table should be printed in the autogenerated excel created when a user click in the export button.
https://issues.liferay.com/browse/LPS-143214

**Proposed Solution**
----------------

Modify the current implementation to add the corresponding id column in the excel generation code.

**Steps to Verify:**
----------------

1. Go to the content dashboard UI
2. Check that some data is already shown in the table (if not, add documents and/or web contents)
3. Click the "Export XLS" button
4. The generated document should contain the same data shown in the content dashboard table and the info panel.

**Change summary:**
----------------

- LPS-143214 Adding id to excel generation
- LPS-143214 Update Excel generation test to also check if the id is present
